module/functions/azure/Get-UniqueSuffix.ps1

```diff
@@ -0,0 +1,51 @@
+# <copyright file="Get-UniqueSuffix.ps1" company="Endjin Limited">
+# Copyright (c) Endjin Limited. All rights reserved.
+# </copyright>
+
+<#
+.SYNOPSIS
+Generates a unique suffix to use for resource names.
+
+.DESCRIPTION
+Generates a deterministic hash from a set of inputs, and returns an 12 character alphabetic string to use as a resource name suffix.
+
+.PARAMETER SubscriptionId
+The subscription ID of the target Azure subscription.
+
+.PARAMETER StackName
+The name of the deployment stack.
```

Comment:
What is this? My guess was that this indicates whether it's the global Azure system, one of the more specialized ones (e.g., the Chinese one, or the German Gov one), or perhaps a private Azure Stack. Jon's guess was that my guess was wrong...

So it would be good to clarify what this actually is, and how I know what to pass in.

Reply:
It's a replacement for what we currently call the rather generic `Prefix`. Essentially a grouping for what may be multiple services (e.g. a Marain stack). Initially I had called it `ApplicationName` but I think this is already a loaded term (and you may have multiple applications within the same deployment).

Reply:
Is this explained somewhere in the comments or in documentation? If not, a copy-paste of the above (with some tweaks) into the script would make it clearer.